app/context.go

```diff
@@ -81,6 +81,10 @@ type context struct {
 	transaction     *transactions.TransactionStore
 	logWriter       io.Writer
 	govupdate       *action.GovernaceUpdateAndValidate
+	extApp          *common.ExtAppData
+	extStores           data.StorageRouter
+	extServiceMap   common.ExtServiceMap
+	controllerFunctions common.Router //External Stores
```

Comment:
Try to format this better, also remove the comment //External Stores next to controllerFunctions, it can be confusing.

Reply:
yeah sure